setup.py

```diff
@@ -25,9 +25,13 @@
       # Extra:
       long_description_content_type="text/markdown",
       setup_requires=["pytest-runner"],
-      tests_require=["pytest"],
+      tests_require=[
+          "vault_dev",
+          "pytest"
+      ],
       requires=[
           "docker",
           "docopt",
+          "hhac",
```

Comment:
Is this a typo? Or is there an hhac as well as an hvac?

Reply:
it's a typo (fixed in 24755db) - I'm surprised nothing in the python packaging complains about this though